cpp/basix/mdspan.hpp

```diff
@@ -1,6 +1,9 @@
 #ifndef _MDSPAN_SINGLE_HEADER_INCLUDE_GUARD_
 #define _MDSPAN_SINGLE_HEADER_INCLUDE_GUARD_
 
+#define MDSPAN_USE_PAREN_OPERATOR 1
```

Comment:
Set these via Cmake with a comment explaining the issue so we will remember to revisit.